dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sme/categories/power/subcategories/SmePowerInvestments.kt

```diff
@@ -0,0 +1,11 @@
+package org.dataland.datalandbackend.model.sme.categories.power.subcategories
+
+import org.dataland.datalandbackend.model.enums.sme.PercentRangeForInvestmentsInEnergyEfficiency
+
+/**
+ * --- API model ---
+ * Fields of the subcategory "Investments" belonging to the category "Power" of the sme framework.
+*/
+data class SmePowerInvestments(
+    val percentageOfInvestmentsInEnhancingEnergyEfficiency: PercentRangeForInvestmentsInEnergyEfficiency? = null,
```

Comment:
Is this a good name for the variable type?

Reply:
We can discuss it. However, they are autogenerated through the new devtool outputs.

Reply:
As the name is aligned with the data dictionary category and sub-category and the naming of these categories is acceptable from my perspective leave it as is.